### PR TITLE
Feat/dob 50 refactor consulting type loading

### PR DESCRIPTION
**Pull request by Schweizer-Philipp**

### CODE REVIEW

**Review by PepeCortisona — APPROVED** [review]

File: services/consultingtypeservice.yaml [hunk 1]
```diff
@@ -0,0 +1,372 @@
+openapi: 3.0.1
+info:
+  title: will be replaced
```
Comment (daho4b) [inline services/consultingtypeservice.yaml:3]: the file has been updated in the service, please update the file here as well

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -42,3 +42,11 @@ csrf.cookie.property=
 
 # UserAdmin service url
 user.admin.service.api.url=
+
+# Cache Configuration
+cache.consulting.type.configuration.maxEntriesLocalHeap=100
+cache.consulting.type.configuration.eternal=false
+cache.consulting.type.configuration.timeToIdleSeconds=0
+cache.consulting.type.configuration.timeToLiveSeconds=10800
```
Comment (daho4b) [inline src/main/resources/application.properties:50]: I would suggest increasing the value for 'timeToLiveSeconds'

File: src/test/java/de/caritas/cob/agencyservice/testHelper/ExtendedConsultingTypeResponseDTOHelper.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+public class ExtendedConsultingTypeResponseDTOHelper {
+
+  public static ExtendedConsultingTypeResponseDTO createExtendedConsultingTypeResponseDTO(int id,
+      WhiteSpot whiteSpot, Registration registration, Boolean isLockedAgencies) {
+
+    var extendedConsultingTypeResponseDTO = new ExtendedConsultingTypeResponseDTO();
+
+    WhiteSpotDTO whiteSpotDTO = null;
+    if (whiteSpot != null) {
```
Comment (daho4b) [inline src/test/java/de/caritas/cob/agencyservice/testHelper/ExtendedConsultingTypeResponseDTOHelper.java:17]: Please replace != null with nonNull(...)

**Review by daho4b — CHANGES_REQUESTED** [review]
See comments

**Review by PepeCortisona — APPROVED** [review]

**Pull request merged**